### PR TITLE
 Use lower case for `@SuppressWarnings` keys

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: checker/src/main/java/org/checkerframework/checker/i18nformatter/qual/I18nConversionCategory.java [hunk 1]
```diff
@@ -75,7 +78,7 @@
      */
     public static I18nConversionCategory stringToI18nConversionCategory(String string) {
         string = string.toLowerCase();
-        for (I18nConversionCategory v : new I18nConversionCategory[] {DATE, NUMBER}) {
+        for (I18nConversionCategory v : namedCategories) {
```
Comment (smillst) [inline checker/src/main/java/org/checkerframework/checker/i18nformatter/qual/I18nConversionCategory.java]: This change isn't related to this pull request.

File: checker/src/main/java/org/checkerframework/checker/i18nformatter/I18nFormatUtil.java [hunk 2]
```diff
@@ -361,6 +360,10 @@ private static void makeFormat(int offsetNumber, StringBuilder[] textSegments) {
             categories.add(category);
         }
 
+        /**
+         * Return the index of s in list. If not found, return the index of
+         * s.trim().toLowerCase(Locale.ROOT) in list. If still not found, return -1.
```
Comment (smillst) [inline checker/src/main/java/org/checkerframework/checker/i18nformatter/I18nFormatUtil.java]: This change isn't related to this pull request.

File: framework/src/main/java/org/checkerframework/common/subtyping/SubtypingChecker.java [hunk 3]
```diff
@@ -23,20 +24,37 @@
...
+     *
+     * <pre>{@code
+     * SubtypingChecker.getSuppressWarningsKeys(this.visitor, super.getSuppressWarningsKeys());
+     * }</pre>
+     *
+     * @param visitor the visitor
+     * @param superSupportedTypeQualifiers the result of super.getSuppressWarningsKeys(), as
+     *     executed by checker
+     */
+    public static Collection<String> getSuppressWarningsKeys(
```
Comment (smillst) [inline framework/src/main/java/org/checkerframework/common/subtyping/SubtypingChecker.java]: This change should be added to the description of this pull request or moved to a separate request.

**Comment by mernst** [comment]
@smillst I pruned this down to a single change.  Could you take another quick look?

**Pull request merged**